packages/starknet-snap/test/src/signMessage.test.ts

```diff
@@ -40,7 +40,7 @@ describe('Test function: signMessage', function () {
   };
 
   const requestObject: SignMessageRequestParams = {
-    chainId: STARKNET_TESTNET_NETWORK.chainId,
+    chainId: STARKNET_SEPOLIA_TESTNET_NETWORK.chainId,
     signerAddress: account1.address,
     typedDataMessage: typedDataExample,
     enableAutherize: true,
```

Comment:
@stanleyyconsensys can you fix this typo`Authorize` instead of `Autherize`

Reply:
lets do it in other branch, this branch is remove network